glide.lock

```diff
@@ -8,9 +8,7 @@ imports:
   subpackages:
   - gogen
 - name: github.com/DataDog/datadog-agent
-  version: a1a945b5c4cbf64ee2762a71c25d6229c57a4840
-  repo: git@github.com:DataDog/datadog-agent.git
-  vcs: git
+  version: 92c410ac3af9d73d237fe076d77d1d365b9abe5c
```

Comment:
https://github.com/DataDog/datadog-agent/commit/92c410ac3af9d73d237fe076d77d1d365b9abe5c

This just rebases https://github.com/DataDog/datadog-agent/pull/847 off of master so I could test with the latest kubelet tagger changes.